AspNetDependencyInjection.Mvc/DependencyInjectionMvcDependencyResolver.cs

```diff
@@ -49,7 +49,7 @@ public IEnumerable<Object> GetServices(Type serviceType)
 			// This implementation from `Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions`:
 			Type closedGenericType = typeof(IEnumerable<>).MakeGenericType( serviceType );
 
-			return (IEnumerable<Object>)this.GetService( serviceType );
+			return (IEnumerable<Object>)this.GetService( serviceType ) ?? Enumerable.Empty<Object>();
```

Comment:
I reviewed the changes, and, on the surface, it appears to work.  However, should somebody actually implement an `IFilterProvider`, or perhaps `ModelBinderProvider`, `ValueProviderFactory`, or any of the other built in types that MVC tries to load via `GetServices`, it will fail as the individual type cannot be cast to `IEnumerable<object>` (unless the type you registered just so happened to also implement `IEnumerable<>`).  Instead, the composed `IEnumerable<>` type should be passed in, which then obtains the list of services properly.  I have tested that doing such works properly.  Should you update the pull request accordingly, I can accept the pull request.
```suggestion
			return (IEnumerable<Object>)this.GetService( closedGenericType ) ?? Enumerable.Empty<Object>();
```